### DISCUSSION TITLE
How to target and style one specific grid line?

### DISCUSSION THREAD

**Discussion by nev3rfind** [apexcharts/apexcharts.js#3119] [Q&A]
I am trying to target one grid line according to the tick amount and style it differently apart from all other grid lines. 
Is it possible to do it?

Desired result:

![image](https://user-images.githubusercontent.com/57880277/165742302-dd1941f5-e5b0-4a88-9d6c-84b49582097a.png)

**Comment by MarioPerini** [comment]
Try annotations:
https://apexcharts.com/javascript-chart-demos/line-charts/line-chart-annotations/
